Content.Shared/SS220/SharedTriggers/SS220SharedTriggerEvent/SS220SharedTriggerSystem.cs

```diff
@@ -0,0 +1,15 @@
+// © SS220, An EULA/CLA with a hosting restriction, full text: https://raw.githubusercontent.com/SerbiaStrong-220/space-station-14/master/CLA.txt
+
+namespace Content.Shared.SS220.SharedTriggers.SS220SharedTriggerEvent;
+
+/// <summary>
+/// public method for raises SS220SharedTriggerEvent
+/// </summary>
+public sealed class SS220SharedTriggerSystem : EntitySystem
+{
+    public void SendTrigger(EntityUid uid, EntityUid user)
```

Comment:
Также не уверен насколько целесообразно обязательно передавать юзера, разве стригеррить предмет без юзера не должно быть возможным?
```suggestion
    public void Trigger(EntityUid uid, EntityUid user)
```